react-native/react/tracker/proofs.render.desktop.js

```diff
@@ -1,77 +1,68 @@
 'use strict'
+/* @flow */
 
+// $FlowIssue base-react
 import React, {Component} from '../base-react'
 
-// TODO const when integrating
-const verified = 'verified'
-const checking = 'checking'
-const deleted = 'deleted'
-const unreachable = 'unreachable'
-const pending = 'pending'
+import {error, pending} from '../constants/tracker'
+import type {SimpleProofState} from '../constants/tracker'
+
+export type Proof = {
+  id: string,
+  type: string,
+  state: SimpleProofState,
+  humanUrl: ?string,
+  name: string,
+  color: string
+}
+
+export type ProofsProps = {
+  proofs: Array<Proof>
+}
 
 export default class ProofsRender extends Component {
-  openLink (proof, platform) {
-    window.open(platform ? proof.platformLink : proof.proofLink)
-  }
+  props: ProofsProps;
 
-  metaColor (pp) {
-    return {
-      'new': 'orange',
-      deleted: 'red',
-      unreachable: 'red',
-      pending: 'gray'
-    }[pp.proof.meta]
+  // TODO hook this up
+  openLink (url: string): void {
+    window.open(url)
```

Comment:
we should likely use shell.openExternal and now window.open (which replaces our window right?) (i know i probably wrote that...)